src/test/java/uk/gov/ons/ctp/response/action/scheduled/distribution/ActionDistributorTest.java

```diff
@@ -85,10 +80,18 @@ public void setUp() throws Exception {
     actionCase = new ActionCase();
     actionCase.setSampleUnitType("H");
 
+    MockitoAnnotations.initMocks(this);
+    DataGrid dataGrid = new DataGrid();
+    dataGrid.setLockTimeToLiveSeconds(30);
+    dataGrid.setLockTimeToWaitSeconds(600);
+    when(appConfig.getDataGrid()).thenReturn(dataGrid);
```

Comment:
alternatively, you could've returned a mock `DataGrid` and intercepted the `getLockTime` methods.